release-please-config.json

```diff
@@ -2,7 +2,7 @@
   "bootstrap-sha": "188faf7e47b03bde3bf02c3521771301c77580d6",
   "packages": {
     ".": {
```

Comment:
This `.` signals that we release (with release please) the whole repo as one release (which means the SDK will bump its number even if we only have changes in the open feature provider).

I guess it would be the easiest approach and maybe we could improve on it later.